utils/mapping.js

```diff
@@ -0,0 +1,81 @@
+import axios from 'axios'
+import stringify from 'fast-json-stable-stringify';
+import network from '@/constant/network';
+import { API_LIKER_NFT_MAPPING } from '../constant/api'
+
+async function signISCNMapping(inputWallet, signer, platform, iscnId, url, likerId) {
+  if (!inputWallet) return null;
+  if (![
+'likeWallet',
+'cosmosWallet',
+].includes(platform)) {
+    throw new Error('SIGN_COSMOS_LOGIN_INVALID_PLATFORM');
+  }
+  const ts = Date.now();
+  const payload = JSON.stringify({
+    ts,
+    iscnId,
+    url,
+    likerId,
+    [platform]: inputWallet,
+  });
+  const {
+    signed: message,
+    signature: { signature, pub_key: publicKey },
+  } = await signer(payload);
+  const data = {
+    signature,
+    publicKey: publicKey.value,
+    message: stringify(message),
+    from: inputWallet,
+    platform,
+  };
+  return data;
+}
+
+async function signLogin(signPayload,signer,address) {
+  if (!window.keplr) throw new Error('CANNOT_FIND_KEPLR');
+  window.keplr.defaultOptions = window.keplr.defaultOptions || {};
+  const originalConfig = window.keplr.defaultOptions.sign || {};
+  window.keplr.defaultOptions.sign = {
+    ...originalConfig,
+    preferNoSetFee: true,
+    preferNoSetMemo: true,
+  };
```

Comment:
Please remove these for now since we might be on wallet connect instead of keplr